build.xml

```diff
@@ -182,52 +182,44 @@
     <!-- Create demo users. -->
     <target name="create-demo-users" description="Create demo users.">
         <foreach list="${drupal.demo.users}" param="drupal.demo.user" target="create-demo-user" delimiter="," />
-        <!-- Create a user with only "authenticated user" role assigned. -->
+        <foreach list="${drupal.demo.authenticated_users}" param="drupal.demo.user" target="create-demo-authenticated_user" delimiter="," />
+    </target>
+
+    <!-- Create a demo user. Subtarget of "create-demo-users". -->
+    <target name="create-demo-user">
```

Comment:
The reason behind this is that before we were creating 3 users. A moderator, an administrator and an authenticated user. The first two were mapped with a role identical to their name so we had an extra target only to create an authenticated user.

Now I have added  two more users that will serve as facilitators for collections and solutions so I converted the second target into a loop as well so that I can add the 3 authenticated users.